app/uk/gov/hmrc/helptosavereminder/models/test/ReminderGenerator.scala

```diff
@@ -28,14 +28,17 @@ import scala.util.Random
 object ReminderGenerator {
   private lazy val rand = new Random()
   private lazy val generator = new Generator(rand)
+  private lazy val ONE_INT = 1
```

Comment:
I don't think that you need to go to this extreme, why not just:

 // scalastyle:ignore magic.number

as elsewhere?

Reply:
Done